src/utils/wrappers.ts

```diff
@@ -164,24 +183,33 @@ export function handleWrappers(ast: Node) {
             return false;
           }
           if (isUndefinedOrVoid(prop.value)) {
-            if (prop.key.type === 'Identifier')
+            if (prop.key.type === 'Identifier') {
               externals[prop.key.name] = {
                 type: 'Literal',
+                // @ts-ignore start can be undefined
                 start: prop.key.start,
+                // @ts-ignore end can be undefined
                 end: prop.key.end,
                 value: prop.key.name,
                 raw: JSON.stringify(prop.key.name)
               };
-            else if (prop.key.type === 'Literal')
-              externals[prop.key.value] = prop.key;
+            } else if (prop.key.type === 'Literal') {
+              externals[String(prop.key.value)] = prop.key;
+            }
           }
         }
         return true;
       })) {
         // if we have externals, inline them into the browserify cache for webpack to pick up
         const externalIds = Object.keys(externals);
-        if (externalIds.length) {
-          const cache = (wrapper.arguments[0].body.body[1] || wrapper.arguments[0].body.body[0]).argument.callee.arguments[1];
+        const argBody = (wrapper.arguments[0].body.body[1] || wrapper.arguments[0].body.body[0]);
+        if (externalIds.length &&
+            argBody &&
+            'argument' in argBody &&
+            argBody.argument &&
+            'callee' in argBody.argument &&
+            'arguments' in argBody.argument.callee) {
+          const cache = argBody.argument.callee.arguments[1];
           cache.properties = externalIds.map(ext => {
             return {
               type: 'Property',
```

Comment:
This whole return type is complaining because it doesn't match Property type. Its missing the following:

```
method: boolean;
shorthand: boolean;
computed: boolean;
```

Also, the nested `CallExpression` is missing the `optional: boolean`.

Is it safe to say these should all be `false`?